helix-core/src/test/resources/TestReplicaLevelThrottling.SingleTopState.json

```diff
@@ -71,6 +71,78 @@
         }
       }
     },
+    {
```

Comment:
Is the fixed logic already tested in the existing test case?

Reply:
Newly added test case covers the fixed logic.